pages.ru/common/head.md

```diff
@@ -0,0 +1,20 @@
+# head
+
+> Выводит первую часть файлов.
+> Больше информации: <https://www.gnu.org/software/coreutils/head>.
+
+- Выводит первые несколько строк из файла:
+
+`head -n {{count_of_lines}} {{filename}}`
```

Comment:
Please translate the placeholders for get this easier for non-English speakers.

Reply:
Sure, one moment.

Reply:
Done.